libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmOpenShiftVersionSelect.tsx

```diff
@@ -72,15 +76,46 @@ const OcmOpenShiftVersionSelect = ({ versions }: OcmOpenShiftVersionSelectProps)
         })),
     [versions, t],
   );
+  const [isOpenshiftVersionModalOpen, setIsOpenshiftVersionModalOpen] = React.useState(false);
+
+  const showOpenshiftVersionModal = () => {
+    setIsOpenshiftVersionModalOpen(true);
+  };
+
+  const setValueSelected = (selectedValue: OpenshiftVersionOptionType) => {
```

Comment:
This function is not necessary, instead of passing `setValueSelected`, it's the same as passing `setSelectedValue`
(but based on my previous comment, I would try to handle everything via the Formik form)